parsers/grammar/camxes.peg

```diff
@@ -94,48 +94,49 @@ I_clause            <- free_prefix* spaces? I                   # 3-words compou
 O_clause            <- free_prefix* spaces? O                   # 4-words compound
 U_clause            <- free_prefix* spaces? U                   # n-words compound starter and terminator
 
-BA_clause           <- free_prefix* spaces? BA                  # abstractors starter
-BAY_clause          <- free_prefix* spaces? BAY free_post*      # abstractors terminator
-BAY_clause_elidible <- BAY_clause?
+PA_clause           <- free_prefix* spaces? PA                  # abstractors starter
+PAY_clause          <- free_prefix* spaces? PAY free_post*      # abstractors terminator
+PAY_clause_elidible <- PAY_clause?
 BQ_clause           <- free_prefix* spaces? BQ                  # letters
```

Comment:
BQ_clause -> VQ_clause?

Reply:
BQ is the letters family. Do you suggest V to be the new first consonant of the alphabet ? :p